packages/marketing-modal/src/blobs.tsx

```diff
@@ -3,13 +3,12 @@ import { palette } from '@leafygreen-ui/palette';
 import { css, cx } from '@leafygreen-ui/emotion';
 import { BlobPosition } from '.';
 
-type SvgBlobProperties = 'viewBox' | 'path' | 'fill' | 'styles';
+type SvgBlobProperties = 'viewBox' | 'path' | 'styles';
```

Comment:
@spark33 do we have opinions on where this file should live? I'm thinking inside of the MarketingModal dir, but what do you think?

Reply:
Since we have the `svgBlobs` function which is essentially a subcomponent (even though it's not capitalized), I'm a fan of it having its own subdirectory and its own types.ts file, where this could live.

Reply:
Yeah I was thinking about this, I wasn't sure if the blobs should be a subcomponent or maybe if this and the other svg should be in a utils folder.

Reply:
but i'm not against making it a subcomponent